modules/op-release-notes-1-8.adoc

```diff
@@ -12,13 +12,198 @@ With this update, {pipelines-title} General Availability (GA) 1.8 is available o
+// Shivam Mukhade
+
+* This update implements `Step` and `Sidecar` overrides for `TaskRun` resources.
+// link:https://github.com/tektoncd/pipeline/pull/4598
+
+* This update adds minimal `TaskRun` and `Run` statuses within `PipelineRun` statuses.
+// link:https://github.com/tektoncd/pipeline/pull/4739
+
+* With this update, the graceful termination of pipeline runs feature is promoted from being a beta feature to being a stable feature. As a result, the previously deprecated `PipelineRunCancelled` status remains deprecated and is planned to be removed in a future release.
+// link:https://github.com/tektoncd/pipeline/pull/4668
+
+* With this update, you can specify the workspace for a pipeline task by using the name of the workspace. This change makes it easier to specify a shared workspace for a pair of `Pipeline` and `PipelineTask` resources. You can also continue to map workspaces explicitly.
+// link:https://github.com/tektoncd/pipeline/pull/4887
+
+* With this update, parameters in embedded specifications are propagated without mutations.
+//https://github.com/tektoncd/pipeline/pull/4845
+
+* With this update, you can specify the required metadata of a `Task` resource referenced by a `PipelineRun` resource by using annotations and labels. This way, `Task` metadata that depends on the execution context is available during the pipeline run.
+// link:https://github.com/tektoncd/pipeline/pull/4834
+
+* This update adds support for object or dictionary types in `params`  and `results` values. This change affects backward compatibilities and sometimes breaks forward compatibility, such as using an earlier client with a later {pipelines-title} version.
+Action required: This update changes the `ArrayOrStruct` structure, which affects the project using the go API as a library.
+// TBD what action?
+// link:https://github.com/tektoncd/pipeline/pull/4786
+
+* This update adds a `SkippingReason` value to the `SkippedTasks` field of the `PipelineRun` status fields so that users know why a given PipelineTask was skipped.
+// link:https://github.com/tektoncd/pipeline/pull/4829
+
+* As an alpha feature, this update supports using an `array` type for emitting results from a `Task` object. The type of the result is changed from `string` to `ArrayOrString`.
++
+A task can specify a type to produce an array result, such as:
++
+[source,yaml]
+----
+kind: Task
+apiVersion: tekton.dev/v1beta1
+metadata:
+  name: write-array
+  annotations:
+    description: |
+      A simple task that writes array
+spec:
+  results:
+    - name: array-results
+      type: array
+      description: The array results
+...
+----
++
+And the task script can populate the result in an array form with:
++
+[source,terminal]
+----
+$ echo -n "[\"hello\",\"world\"]" | tee $(results.array-results.path)
+----
++
+This feature is part of the TEP-0076 and is in progress to index into the array result while consuming that result.
```

Comment:
```suggestion
This feature is in progress and is part of TEP-0076.
```
This part of the sentence doesn't make sense to me as constructed: `the feature to index into the array result while consuming that result` -- how can you index INTO a result?  Could you seek clarification? Or is it enough to just cut out that part?

Reply:
Thanks. I've applied your suggestion.